tenacity/__init__.py

```diff
@@ -279,7 +279,9 @@ def copy(self, sleep=_unset, stop=_unset, wait=_unset,
             before=self.before if before is _unset else before,
             after=self.after if after is _unset else after,
             before_sleep=before_sleep,
-            reraise=self.reraise if after is _unset else reraise,
```

Comment:
This also looked like a copy paste bug, since `reraise` is also passed in to the copy function.